ast/test/model.go

```diff
@@ -1,8 +1,11 @@
 package user
 
+import "time"
+
 type User struct {
-	Name  string
-	Admin bool `field: yolo`
```

Comment:
Should this be `field:"yolo"` or was this intentional?

Reply:
It should be `field:"yolo"`. [StructTag docs describe the convention](https://pkg.go.dev/reflect#StructTag)